modify java.md, python.md, and configure-introduction.md (#260)
tikv/website: content/docs/5.1/deploy/configure/introduction.md

        weight: 2
---

> It's recommended to get started with TiKV with default configurations, however, TiKV features a large number of configuration parameters to tweak its behavior, allowing you to configure the cluster to suit the special application requirements.

Reviewer: It's --> It is 
Please spell it in full.

> It's recommended to get started with TiKV with default configurations, however, TiKV features a large number of configuration parameters to tweak its behavior, allowing you to configure the cluster to suit the special application requirements.

Reviewer: ```suggestion
It is recommended to get started with TiKV with default configurations. However, TiKV features a large number of configuration parameters to tweak its behavior, allowing you to configure the cluster to suit your special application requirements.
```

> It is recommended to get started with TiKV with default configurations. However, TiKV features a large number of configuration parameters to tweak its behavior, allowing you to configure the cluster to suit your special application requirements.

Reviewer: ```suggestion
Although you are recommended to get started with TiKV using the default configuration, TiKV provides many configuration parameters to tweak its behavior, which allows you to configure the cluster to suit your special application requirements.
```

> The following guides provide you ways to configure different TiKV components:

Reviewer: ```suggestion
The following documents guide you on how to configure different TiKV components:
```

> The following documents guide you on how to configure different TiKV components:

Reviewer: ```suggestion
The following list of documents guide you on how to configure different TiKV components:
```

> The following list of documents guide you on how to configure different TiKV components:

Reviewer: ```suggestion
The following lists of documents guide you on how to configure different TiKV components:
```

> - [Region Merge](../region-merge): Tweak region merging.

Reviewer: ```suggestion
- [Region Merge](../region-merge): Tweak Region merging.
```
